go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/99designs/gqlgen
 
+go 1.12
```

Comment:
Sure?

Reply:
1.13 [just released](https://golang.org/doc/go1.13)

Reply:
AFAIK, go 1.12 will make gqlgen compatible with any users not on 1.13